pcbdraw/pcbdraw.py

```diff
@@ -848,6 +928,8 @@ def main():
     parser.add_argument("--dpi", help="DPI for bitmap output", type=int, default=300)
     parser.add_argument("--no-warn-back", action="store_true", help="Don't show warnings about back footprints")
     parser.add_argument("--shrink", type=float, help="Shrink the canvas size to the size of the board. Specify border in millimeters")
+    parser.add_argument("--resistor-values", help="A comma seperated list of what value to set to each resistor for the band colors. For example, \"R1:10k,R2:470\"")
+    parser.add_argument("--resistor-flip", help="A comma seperated list of throughole resistors to flip the bands")
```

Comment:
This is more-less a philosophical question of whether we should supply these values as arg or from a file. If I want to generate a drawing of a board, I usually want to preserve the setting so I would like to put it into a file. Then I have to invoke it like: `pcbdraw --resistor-values $(cat myfile)`.

I am probably fine with keeping this as it is. I am considering switching to [click](https://click.palletsprojects.com/en/8.0.x/) for the v1 release and then we can address this. Also, as KiKit features schematics parser, we could also supply the values directly from the schematics.